nibabel/tests/test_helpers.py

```diff
@@ -42,15 +41,12 @@ def bz2_mio_error():
     This won't cause a problem for scipy releases after Jan 24 2014 because of
     commit 98ef522d99 (in scipy)
     """
-    try:
-        import scipy.io
-    except ImportError:
-        return True
     with InTemporaryDirectory():
         with Opener('test.mat.bz2', 'wb') as fobj:
             try:
+                import scipy.io
```

Comment:
OK. In this case, I would just do the `optionalpackage` check. I thought the combined `try...catch` was more compact and easy to read, but glad to change.